sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_Healthcare.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples: SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public async Task Healthcare()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            #region Snippet:TextAnalyticsSample1CreateClient
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            #endregion
+
+            #region Snippet:Healthcare
```

Comment:
The snippet names need to be generic in the repo, so I will recommend following what we are doing in the other TA samples: `Snippet:TextAnalyticsSampleHealthcare`

More info: https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets